src/main/resources/liquibase/scripts/scriptsBot.sql

```diff
@@ -2,7 +2,9 @@
 
 -- changeset krassotin:1
 create table notification_task(
-                                  notification_task_id serial primary key ,
+
+                                  task_id serial primary key ,
                                   chat_id serial not null ,
```

Comment:
Тут тоже надо поменять тип:
```suggestion
                                  chat_id int8 not null ,
```